assets/src/edit-story/components/panels/backgroundFill.js

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { BACKGROUND_FILL_TYPES } from '../../constants';
+import { SimplePanel } from './panel';
+
+function BackgroundFillPanel({ selectedElements, onSetProperties }) {
+  const { backgroundFill } = selectedElements[0];
+
+  const handleChange = (evt) => {
+    const newValue =
+      evt.target.value === BACKGROUND_FILL_TYPES.DO_NOT_FORMAT
```

Comment:
One nit: I really have hard time with the "Do not format" vs "Fit to device" naming. I wonder if we should just call it "isFullbleed" property - that's what it really is. Those names are just trying to explain this concept to the designer. But I'd not be surprised if we would be trying different namings for awhile, while the concept stays the same.

Reply:
Makes sense 👍